sklearn/metrics/_regression.py

```diff
@@ -188,6 +190,81 @@ def mean_absolute_error(y_true, y_pred,
     return np.average(output_errors, weights=multioutput)
 
 
+def mean_absolute_percentage_error(y_true, y_pred,
+                                   sample_weight=None,
+                                   multioutput='uniform_average'):
+    """Mean absolute percentage error regression loss
+
+    Note here that we do not represent the output as a percentage in range
+    [0, 100]. Instead, we represent it in range [0, 1]. Read more in the
```

Comment:
Surely the value is bounded from above by 1/eps, not 1?

Reply:
done